src/clj/puppetlabs/services/jruby/puppet_environments.clj

```diff
@@ -36,4 +37,7 @@
       (environment-state [this] state)
       (mark-all-environments-expired! [this]
         (log/info "Marking all registered environments as expired.")
-        (swap! state mark-all-expired!)))))
+        (swap! state mark-all-expired!))
+      (mark-environment-expired! [this env-name]
+        (log/infof "Marking environment '%s' as expired." env-name)
+        (swap! state mark-expired! (keyword env-name))))))
```

Comment:
That's a good point.  It seems like this endpoint should return an error in the case when the specified environment doesn't exist.